test/build_openssl.sh

```diff
@@ -33,6 +34,8 @@ case "${TARGET}" in
     ;;
 esac
 
+apt-get install -y --no-install-recommends curl
```

Comment:
Shouldn't this logic (and most of the logic in this script) end up getting removed in favor of the src crate?